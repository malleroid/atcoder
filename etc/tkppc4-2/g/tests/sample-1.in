3 200
2804
2416 20 30 80
2269 80 20 80
2115 30 40 70
