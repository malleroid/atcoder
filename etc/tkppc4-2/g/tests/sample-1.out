2656.33333
